### PR TITLE
ADA Tasks

### PR DESCRIPTION
**Pull request by fditata**

### CODE REVIEW

File: src/components/contactForm.js [hunk 1]
```diff
@@ -130,20 +128,26 @@ const ContactForm = () => {
...
               required={true}
             />
           </div>
-
           <button className="btn btn-md primary min-130" type="submit">
             {contactFormBtn}
           </button>
         </form>
+        <div id="infoDiv">
+            {/* ... Info related to the inquiryType ... */}
```
Comment (nonahayrapetyan) [inline src/components/contactForm.js:143]: what this empty div are for?

File: src/pages/calculator-no-pets-allowed.js [hunk 2]
```diff
@@ -186,6 +186,23 @@ const CalculatorNoPetsAllowed = () => {
...
+        emptyKeys.push(key);
+      }
+    });
+  
+    return emptyKeys;
+  };
+  
+  const emptyConstants = checkEmptyValues(useWithNoPetCalculatorData());
+  
+  console.log("Empty constants are:", emptyConstants);
```
Comment (nonahayrapetyan) [inline src/pages/calculator-no-pets-allowed.js:203]: remove console

File: src/pages/calculator-no-pets-allowed.js [hunk 3]
```diff
@@ -186,6 +186,23 @@ const CalculatorNoPetsAllowed = () => {
...
+    Object.entries(data).forEach(([key, value]) => {
+      if (!value) {
+        emptyKeys.push(key);
+      }
+    });
+  
+    return emptyKeys;
+  };
+  
+  const emptyConstants = checkEmptyValues(useWithNoPetCalculatorData());
```
Comment (nonahayrapetyan) [inline src/pages/calculator-no-pets-allowed.js:201]: why do we need this if is not used?

Comment (fditata) [reply src/pages/calculator-no-pets-allowed.js:201]: I used to check the empty vars. I removed it now.

File: src/components/contactForm.js (hunk 1 above)
Comment (fditata) [reply src/components/contactForm.js:143]: This comes from ADA issues. I think is a false positive. Usually that div should have content, but I don't think is necessary. I just want to try if doing this removes the issue about accessibility.

**Pull request closed without merging**